build.sh

```diff
@@ -12,6 +12,10 @@ BUILD_TIME=$(date +%Y%m%d-%H%M)
 BRANCH=image-${BUILD_TYPE}
 PENDING_PRS=${PWD}/pending-prs-${BUILD_TYPE}.json
 BUILD_ID=${BUILD_ID:-}
+LOCAL_REGISTRY=${LOCAL_REGISTRY:-}
+LOCAL_REGISTRY_PASS=${LOCAL_REGISTRY_PASS:-}
+BUILD_HOST=${BUILD_HOST:-}
+BUILD_HOST_USER=${BUILD_HOST_USER:-}
```

Comment:
These lines don't have any effect, maybe you want the entire list here for some reason?

Reply:
They're here for two reasons:

1) I use the so-called "bash strict mode", which would error out if I have undefined variables, so these lines are mandatory if I want to use specific env vars. It's basically being explicit about the usual implicit bash behavior of "undefined variable = empty string".

2) Having the list of env vars used in the file is useful to make sure you have all of them.

Reply:
oh [this](https://github.com/container-mgmt/cm-integration/pull/7/files#diff-0b83f9dedf40d7356e5ca147a077acb4R7)